wen-restart/src/wen_restart.rs

```diff
@@ -473,6 +480,10 @@ pub(crate) fn aggregate_restart_heaviest_fork(
     let mut progress_last_sent = Instant::now();
     let mut cursor = solana_gossip::crds::Cursor::default();
     let mut progress_changed = false;
+    let majority_stake_required = (total_stake as f64 / 100.0
+        * (wait_for_supermajority_threshold_percent as f64
+            - HEAVIEST_FORK_DISAGREE_THRESHOLD_PERCENT))
+        .round() as u64;
```

Comment:
can we just reuse `adjusted_threshold_percent` here

Reply:
Done